semana4/lista-de-exercicios/index.js

```diff
@@ -0,0 +1,327 @@
+//AULA 14
+
+/*leitura de codigo
+exercicio 1
+mostra um codigo de conversão de dinheiro de dolar para real, uma function
+para receber o valorem dolar e devolver com o preço convertido em real,
+aplicando a conversão
+
+exercicio 2
+um function que recebe dois paramentros que entra em um switch para escolher o tipo de investimento
+e quanto quer aplicar, apos ser identificado pelo case, é feito aplicação do novo montante e o 
+segundo montante sendo mostrado no console.log
+
+exercicio 3
+são declarados 3 arrays um chamado numeros que possue valores dentro e os outros dois vazios, if os
+valores encontrados no vetor numero for par, essse valor será inteirado no array1, caso seja impar
+será parte do array2
+
+exercicio 4
+é declarado um vetor com varios valores tipo number, mais dois valores do tipo let com os nomes de 
+numero1 e numero2, apos isso entra em um laço(for) que percorre todo o vetor numeros, si o elemento
+numero do vetor for menor que a variavel numero1 então numero1 recebe numero e if numero maior numero2
+entao numero2 recebe numero,  apos disso é printado no console
+
+exercicios de logica de promgramação
+
+exercicio 1
+a) false
+b) false
+c) true
+d) true
+e) false
+
+//exercicio 2
+
+ele executa, porem nao necessariamente vai vir numeros pares, mas sim I multiplicado por 2, ate que
+saia do laço.
+
+const quantidadeDeNumerosPares 
+let i = 0
+let somador = 0
+while(i <quantidadeDeNumerosPares){
+    if(somador%2 === 0){
+        console.log(somador)
+        somador ++
+        i++
+    }else{
+        somador ++
+    }
+}
+
+//exercicio 3
+function trigonometria(a,b,c){
+    if(a===b && b==c){
+        return console.log("triângulo Equilátero")
+    }
+    if(a===b && a != c ){
+        return console.log("triângulo Isóceles")
+    }else if(b==c && b != a ){
+        return console.log("triângulo Isóceles")
+    }else if(a==c && a!==b){
```

Comment:
Você poderia colocar essas três verificações do triangulo isóceles em um else if só, bastaria usar os parênteses dentro dos parênteses. A sintaxe ficaria mais ou menos assim:

`else if ((a===b && a !== c ) || (b==c && b !== a) || ((a==c && a!==b)) {return "isoceles"}`